.github/workflows/reusable_terraform_plan_apply.yml

```diff
@@ -126,6 +126,10 @@ jobs:
             const identifier = workflowId ? `_${workflowId}_\n` : `_${{ inputs.workflow_id }}_\n`;
             const summary = `\`${{ steps.show.outputs.summary }}\``;
             const issue_number = context.payload.pull_request.number;
+            if (!issue_number) {
```

Comment:
I don't see how this can work. You are comparing null to boolean and I cannot see the evidence of it working, as this run has a PR number.